lang/de/miquiz.php

```diff
@@ -5,31 +5,36 @@
 $string['modulename'] = 'MI-Quiz';
 $string['modulenameplural'] = 'MI-Quizze';
 
-//$instance_name ='';
 $instance_name = get_config('mod_miquiz', 'instancename');
 $string['modulename_help'] = 'Erstelle ein ' . $instance_name . ' Duell.';
 $additional_info = get_config('mod_miquiz', 'additional_info');
 if (!empty($additional_info)) {
   $string['modulename_help'] .= $additional_info;
 }
 
+//strings for Administration->Manage roles page 
+$string['miquiz:overview'] = 'Globale ' . $instance_name . ' Aktivitäts&uuml;bersicht';
+$string['miquiz:addinstance'] = 'Erstelle ein ' . $instance_name . ' Duell';
```

Comment:
Instead of "Duell" either "Aktivität" or "Quiz", but with the instance_name something like "SKILL-Quiz" "Aktivität" would be better.

Reply:
I meant `$string['miquiz:addinstance']`, but `$string['modulename_help']` needed to changed as well. So now only `$string['miquiz:addinstance']` needs to be changed